docs/source/manual/rendering.rst

```diff
@@ -0,0 +1,40 @@
+.. _rendering:
+
+==================
+Rendering Geometry
+==================
+
+There are many reasons to render geometry beyond the VTK based
+pyg44omety viewer. Potential targets are 3D asset packages (e.g.
+blender, houdini), games engines (unreal engine, unity). A common
+format is Universal Scene Description (USD). USD can be loaded in
+many packages and so a dedicated UDS converter/viewer is available
+in pyg4ometry. For example to load and convert a file to USD
+
+  .. code-block:: python
+    :linenos:
+
+    r = pyg4ometry.gdml.Reader("lht.gdml")
+    l = r.getRegistry().getWorldVolume()
+    v = pyg4ometry.visualisation.UsdViewer("lht.usd")
+    v.traverseHierarchy(l)
+    v.save()
+
+This will write a file called lht.gdml which can be viewed in
+a USD viewer like usdview. USD has a preview shader. The settings
+can set via the `VisualisationOptions` stored in logical volumes
+(`visOption`), so for example
+
+  .. code-block:: python
+    :linenos:
+
+    r = pyg4ometry.gdml.Reader("lht.gdml")
+    reg = r.getRegistry()
+    reg.logicalVolumeDict["vCalorimeter"].visOptions.colour = [1, 0, 0]
+    v = pyg4ometry.visualisation.UsdViewer("lht.usd")
+    v.traverseHierarchy(reg.getWorldVolume())
+    v.save()
+
+`visOptions.color` and `visOptions.alpha` are passed though to the
```

Comment:
This should be `visOptions.colour` instead of `visOptions.color`?

I think this would be important to fix in the docs, to not confuse users after the next release (it already is confusing enough that pyg4ometry uses colour and USD uses ...Color)

Reply:
Fixed.